apps/nextjs/src/app/tokenomics/Dashboard.tsx

```diff
@@ -87,6 +104,19 @@ export const DashBoard = ({
     {},
   );
 
+  const galleonAddress = stakingAddresses[NETWORK_NAME].v1Galleon;
+  const carrackAddress = stakingAddresses[NETWORK_NAME].v2Carrack;
+
+  const totalStakedRealmsData = await getWalletRealmsHeld({
```

Comment:
This should probably be moved to the page.tsx server component and passed as props (and async removed from the root function - which is not allowed in client components). In the future we will should move to a better hydration process with react-query